services/housetables/build.gradle

```diff
@@ -20,6 +20,13 @@ plugins {
   id 'openhouse.service-specgen-convention'
 }
 
+configurations {
```

Comment:
Can we check which dependencies are introducing these modules? if for example, say ` project(':cluster:storage') ` is introducing both, it would be better removed at that layer. Otherwise, it might pop up again in another situation.

Reply:
you can use a command as follows to investigate:
```
./gradlew :module:dependencyInsight --dependency checkerframework --configuration compileClasspath

Reply:
Added to :cluster module since slf4j:log4j was pulled from there. 
Also, leaving the exclusions in :services:housetables for added safety.